discord/constants.py

```diff
@@ -19,42 +25,14 @@
     'targets': {'name': 'targets destroyed', 'value': rwr.constants.PlayersSort.TARGETS_DESTROYED.value, 'getter': lambda player: player.targets_destroyed_display},
 }
 
-VALID_EVOLUTION_TYPES = {
-    'ratio': {'name': 'K/D ratio', 'column': 'kd_ratio'},
-    'score': {'name': 'score', 'column': 'score'},
-    'position': {'name': 'position (by XP)', 'column': 'leaderboard_position'},
-}
-
-VALID_EVOLUTION_TYPE_NAMES = VALID_EVOLUTION_TYPES.keys()
-
-VALID_SERVER_TYPES = {
-    'vanilla': 'vanilla',
-    'pacific': 'pacific',
-    'rwd': 'Running_with_the_Dead'
-}
+DATABASE_CHOICES = Enum('DATABASE_CHOICES', {d: d for d in rwr.constants.VALID_DATABASES})
+EVOLUTION_TYPE_CHOICES = Enum('EVOLUTION_TYPE_CHOICES', {d: d for d in EVOLUTION_TYPES.keys()})
+SERVER_TYPE_CHOICES = Enum('SERVER_TYPE_CHOICES', {d: d for d in rwr.constants.VALID_SERVER_TYPES})
+PLAYER_SORT_CHOICES = Enum('PLAYER_SORT_CHOICES', {d: d for d in PLAYER_SORTS.keys()})
 
-EMBED_COLOR = 10800919 # The well-known primary RWRS color #A4CF17, in the decimal format
+DEFAULT_DATABASE = DATABASE_CHOICES.invasion
+DEFAULT_PLAYER_SORT = PLAYER_SORT_CHOICES.score
 PLAYERS_LIMIT = 24
 SERVERS_LIMIT = 10
-
-AVAILABLE_COMMANDS = OrderedDict([
-    ('stats', {'description': 'Displays stats about a given player', 'admin_only': False}),
-    ('top', {'description': 'Displays the top 24 players', 'admin_only': False}),
-    ('pos', {'description': 'Highlights the given player in the leaderboard', 'admin_only': False}),
-    ('compare', {'description': 'Compare stats of two players', 'admin_only': False}),
-    ('evolution', {'description': 'Displays the evolution of the specified stat', 'admin_only': False}),
-    ('whereis', {'description': 'Displays information about the server the given player is playing on', 'admin_only': False}),
-    ('server', {'description': 'Displays information about the given server', 'admin_only': False}),
-    ('servers', {'description': 'Displays the first 10 currently active servers with room', 'admin_only': False}),
-    ('now', {'description': 'Displays numbers about the current players and servers', 'admin_only': False}),
-    ('info', {'description': 'Displays information about the bot', 'admin_only': False}),
-    ('say', {'description': 'Makes the bot to say something', 'admin_only': True}),
-    ('cc', {'description': 'Clears the RWRS cache', 'admin_only': True}),
-    ('maintenance', {'description': 'Enables or disables the maintenance mode', 'admin_only': True}),
-    ('motd', {'description': 'Sets or removes the MOTD', 'admin_only': True}),
-    ('event', {'description': 'Sets or removes the next RWR event', 'admin_only': True}),
-    ('user', {'description': 'Performs actions on a specific user', 'admin_only': True})
-])
-
-AVAILABLE_COMMANDS_NAMES = AVAILABLE_COMMANDS.keys()
-AVAILABLE_PUBLIC_COMMANDS_NAMES = [name for name, info in AVAILABLE_COMMANDS.items() if not info['admin_only']]
+MY_DISCORD_ID = '66543750725246976'
```

Comment:
Create dict of discord IDs mapped with usernames, and reuse this dict in the future helper function which returns permissions list